server/services/interventionsService.ts

```diff
@@ -126,14 +126,13 @@ export interface InterventionsFilterParams {
   maximumAge?: number
 }
 
-export interface DraftActionPlan extends ActionPlanFields {
+export interface ActionPlan {
```

Comment:
Perhaps you don't consider it part of the scope of this PR, but I've realised that we don't have any contract tests that show a `GET /action-plan` where the fields are populated. Perhaps we could update the `it('returns an existing submitted action plan')` test with examples of these fields?

Reply:
can we do this in a follow up PR? this is blocking some other bits so i'm keen to get it merged.

Reply:
Yeah, do you mind doing that follow up PR?

Reply:
But could we squash the commits now though? Happy to approve then

Reply:
actually upon inspection there is a test for this


```it('returns an existing submitted action plan', async () => {
      const actionPlanId = '7a165933-d851-48c1-9ab0-ff5b8da12695'

      await provider.addInteraction({
        state: `an action plan exists with ID ${actionPlanId}, and it has been submitted`,
        uponReceiving: `a GET request to view the action plan with ID ${actionPlanId}`,
        withRequest: {
          method: 'GET',
          path: `/action-plan/${actionPlanId}`,
          headers: { Accept: 'application/json', Authorization: `Bearer ${token}` },
        },
        willRespondWith: {
          status: 200,
          body: Matchers.like({
            id: actionPlanId,
            submittedAt: '2021-03-09T15:08:38Z',
          }),
          headers: {
            'Content-Type': 'application/json',
          },
        },
      })```

Reply:
i have squashed as requested

Reply:
misread your comment - i will change the existing test to chekc for populated fields.

Reply:
@mahalrmoj i've updated the contracts in this PR. they should still pass on the backend. would you mind just validating that against this branch? thanks!